src/layout/Main.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography, Box, Grid, IconButton } from '@mui/material';
+import SortIcon from '@mui/icons-material/Sort';
+import Article from '../components/Article';
+import DUMMY_DATA from '../data/data.json';
+
+function Main() {
+  return (
+    <Box className='p-8'>
+      <Box className='flex sm:justify-between sm:flex-row flex-col text-center'>
+        <Typography variant='h6'>Showing 3 of 100 Total Tools.</Typography>
```

Comment:
can u hook it up to the data source (DUMMY_DATA) ?

Reply:
u mean by no. of tools in dummy data ryt?